packages/semantic-ui/package.json

```
@@ -0,0 +1,126 @@
+{
+  "name": "@rjsf/semantic-ui",
+  "version": "2.0.0-alpha.1",
+  "module": "dist/rjsf-semantic-ui.esm.js",
+  "description": "Semantic UI theme, fields and widgets for react-jsonschema-form",
+  "lint-staged": {
+    "{src,test}/**/*.js": [
+      "eslint --fix",
+      "prettier --write",
+      "git add"
+    ]
+  },
+  "main": "lib/index.js",
+  "files": [
+    "dist",
+    "lib"
+  ],
+  "engineStrict": false,
+  "engines": {
+    "npm": ">=2.14.7",
+    "node": ">=6"
+  },
+  "scripts": {
+    "build:lib": "rimraf lib && cross-env NODE_ENV=production babel -d lib/ src/",
+    "build:dist": "rimraf dist && cross-env NODE_ENV=production",
```

```suggestion
    "build:dist": "rimraf dist && cross-env NODE_ENV=production babel -d dist/ src/",
```

Doesn't this script line need something more, like a `babel` command.